test/test_entrypoint.py

```diff
@@ -27,8 +33,39 @@ def test_trivial_argparse():
 
 def test_mocked_cc_pipeline(tmpdir):
     args = parse_args([str(tmpdir), '-a', 'aarch64', '-o', 'ubuntu'])
-    with patch('ros_cross_compile.ros_cross_compile.DockerClient', Mock()) as docker_mock:
+    with patch('ros_cross_compile.ros_cross_compile.DockerClient', Mock()) as docker_mock,\
+            patch('ros_cross_compile.ros_cross_compile.check_install_script_exists') as sc_mock:
         cross_compile_pipeline(args)
+        assert sc_mock.called
         assert docker_mock.called
         assert docker_mock().build_image.call_count == 2
         assert docker_mock().run_container.call_count == 2
+
+
+def test_install_rosdep_script_exist(tmpdir):
+
+    contents = """
+
+"""
+    ws = Path(str(tmpdir))
+    data_dir = ws / 'cc_internals' / 'aarch64-ubuntu-dashing'
+    data_file = data_dir / 'install_rosdeps.sh'
+    data_file.parent.mkdir(parents=True)
+    data_file.write_text(contents)
+    args = parse_args([str(tmpdir), '-a', 'aarch64', '-o', 'ubuntu'])
```

Comment:
it's not necessary to do argument parsing here, you can just pass the tmpdir straight to the helper function. same in other test